crates/miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -16,6 +16,8 @@ const.ERR_EPILOGUE_EXECUTED_TRANSACTION_IS_EMPTY="executed transaction neither c
 
 const.ERR_AUTH_PROCEDURE_CALLED_FROM_WRONG_CONTEXT="auth procedure had been called from outside the epilogue"
 
+const.ERR_EPILOGUE_NONCE_CANNOT_BE_0="nonce cannot be 0 after the first transaction"
```

Comment:
```suggestion
const.ERR_EPILOGUE_NONCE_CANNOT_BE_0="nonce cannot be 0 after an account-creating transaction"
```

Nit/Optional: I think this might be slightly clearer than "first transaction", which could also be read as the order of a transaction in a list of transactions.